thrift/zipkinCore.thrift

```diff
@@ -198,6 +197,17 @@ const string HTTP_RESPONSE_SIZE = "http.response.size"
  */
 const string LOCAL_COMPONENT = "lc"
 
+#***** Annotation.value or BinaryAnnotation.key ******
+/**
+ * When an annotation value, this indicates when an error occurred. When a
+ * binary annotation key, the value is a human readable message associated
+ * with an error.
+ *
+ * Note that RPC spans often include both client and server hosts. It is
+ * possible that only one side perceived the error.
+ */
+const string ERROR = "error"
```

Comment:
ok, so you are saying that the binary annotation shouldn't be present
unless it is critical (i.e. not transient). I can understand why, as for
example this could otherwise stain a recovered span.